### PR TITLE
chore: update css lectures intros

### PR DESCRIPTION
**Pull request by Ksound22**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

**Comment by Dario-DC** [comment]
Hey @Ksound22 you have the changes from your previous PR in this one too.

**Comment by Ksound22** [comment]
> Hey @Ksound22 you have the changes from your previous PR in this one too.

Should be fine now

File: client/i18n/locales/english/intro.json [hunk 1]
```diff
@@ -2064,13 +2064,13 @@
...
         "intro": [
-          "In these lecture videos, you will learn about the fundamentals of user interface (UI) design. You will learn about the terms you need to know to communicate with designers, visual heirarchy, scaling, alignment, whitespace, and many more."
+          "In these lecture videos, you will learn about the fundamentals of user interface (UI) design. You will learn about the terms you need to know to communicate with designers, visual heirarchy, scaling, alignment, whitespace, and much more."
         ]
       },
       "lecture-user-centered-design": {
         "title": "User-Centered Design",
         "intro": [
-          "In these lecture videos, you will learn about best practices necessary in designing user-facing features like dark mode, breadcrumbs, modal dialogs, and many more. You will also learn how to coduct user research, user requirements and testing."
+          "In these lecture videos, you will learn about best practices necessary in designing user-facing features like dark mode, breadcrumbs, modal dialogs, and much more. You will also learn how to coduct user research, user requirements and testing."
```
Comment (ojeytonwilliams) [inline client/i18n/locales/english/intro.json:2073]:
```suggestion
          "In these lecture videos, you will learn about best practices for designing user-facing features like dark mode, breadcrumbs, modal dialogs, and much more. You will also learn how to conduct user research, user requirements and testing."
```
"necessary" implies you can't design without them, but you can. The design will likely be bad, but you can do that.

File: client/i18n/locales/english/intro.json [hunk 2]
```diff
@@ -2371,7 +2371,7 @@
       "lecture-understanding-how-to-work-with-floats-and-positioning-in-css": {
         "title": "Understanding How to Work with Floats and Positioning in CSS",
         "intro": [
-          "In these lecture videos, you will learn how to use CSS positioning and floats. You will learn about position properties like <code>absolute</code>, <code>relative</code>, <code>fixed</code>, and <code>sticky</code>. YOu will also learn about the <code>z-index</code> property."
+          "In these lecture videos, you will learn how to use CSS positioning and floats. You will learn how to use <code>absolute</code>, <code>relative</code>, <code>fixed</code>, and <code>sticky</code> positioning. You will also learn about the <code>z-index</code> property."
```
Comment (ojeytonwilliams) [inline client/i18n/locales/english/intro.json:2374]:
```suggestion
          "In these lecture videos, you will learn how to use CSS positioning and floats. You will learn about <code>absolute</code>, <code>relative</code>, <code>fixed</code>, and <code>sticky</code> positioning. You will also use the <code>z-index</code> property."
```
Too much repetition made it feel clumsy.

File: client/i18n/locales/english/intro.json [hunk 3]
```diff
@@ -2433,7 +2433,7 @@
       "lecture-working-with-css-variables": {
         "title": "Working with CSS Variables",
         "intro": [
-          "In these lecture videos, you will learn how to define and use custom properties, AKA CSS variables. You will also learn about the <code>@property</code> rule and how it works."
+          "In these lecture videos, you will learn how to define and use custom properties, or CSS variables. You will also learn about the <code>@property</code> rule and how it works."
```
Comment (ojeytonwilliams) [inline client/i18n/locales/english/intro.json:2436]:
```suggestion
          "In these lecture videos, you will learn how to define and use custom properties (also known as CSS variables). You will also learn about the <code>@property</code> rule and how it works."
```
This makes it clear that CSS variables and custom properties are one and the same.

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

**Review by ojeytonwilliams — APPROVED** [review]
LGTM 👍

**Review by naomi-lgbt — APPROVED** [review]

**Pull request merged**